client/src/components/radiobuttons/index.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+import '../../App.css';
+import './radiobutton.css'
+
+class Radiobutton extends Component {
+  render() {
+    return (
+      <form className='radiobuttons-style'>
+        <input className='radioclass' type="radio" name="status" value='Cancled' /><label className='label-style'>Cancled</label>
```

Comment:
What happens when this radiogroup's value is changed?

Reply:
we will get the value from the group and save it in DB ,
is that your question?